packages/jmix-front-generator/src/building-blocks/stages/writing/pieces/i18n.ts

```diff
@@ -1,7 +1,8 @@
-import {MemFsEditor} from "yeoman-generator";
 import {Locale} from "../../../../common/model/cuba-model";
 import path from "path";
 import {capitalizeFirst, splitByCapitalLetter} from "../../../../common/utils";
+import { JSONSchema7Type } from "json-schema";
```

Comment:
Why do we need this type? Is there a reason `Record<string, string>` won't suffice?

Reply:
Now `gen.fs.readJSON()` with updated types return this type. If we sign it Record<string, string>, it will be a cast.